packages/app/src/app/components/CreateNewSandbox/CreateSandbox/CreateSandbox.tsx

```diff
@@ -220,12 +166,6 @@ export const CreateSandbox: React.FC<CreateSandboxProps> = props => {
             !rProps.hidden && (
               <div {...rProps}>
                 <MobileTabs>
-                  <Tab {...tab} stopId="Create">
-                    Create Sandbox
-                  </Tab>
```

Comment:
These were causing issues when registering tabs. Small screens will get new tabs in the future.